proto/regen/data/v1/types.proto

```diff
@@ -151,20 +151,25 @@ message ContentEntry {
   // hash is the content hash of the anchored data.
   ContentHash hash = 1;
 
-  // iri is the content IRI of the anchored data.
+  // iri is the IRI of the anchored data.
   string iri = 2;
 
-  // timestamp is the Timestamp of the anchored data.
+  // timestamp is the timestamp of the anchored data.
```

Comment:
```suggestion
  // timestamp of the anchored data.
```

Reply:
This is not the convention for writing comments. We always start with the name of the field.